crates/sui-framework/packages/sui-framework/sources/test/test_scenario.move

```diff
@@ -95,6 +109,106 @@ public struct TransactionEffects has drop {
     num_user_events: u64,
 }
 
+//
+// `TxContextBuilder` api
+//
+
+/// Create a new `TxContextBuilder` with the given `sender` address.
+/// Also provides default for all other fields.
+public fun new_tx_context_builder(sender: address): TxContextBuilder {
+    TxContextBuilder {
+        sender,
+        epoch: 0,
+        epoch_timestamp_ms: 0,
+        ids_created: 0,
+        rgp: option::some(700),
+        gas_price: 1_000,
+        gas_budget: 100_000_000,
+        sponsor: option::none(),
+    }
+}
+
+/// Create a `TxContextBuilder` from an existing `TxContext` in a `Scenario`.
+public fun builder_from_scenario(scenario: &Scenario): TxContextBuilder {
```

Comment:
I feel like this and the one below should be prefixed by `tx_context_` possibly? Otherwise the name feels ambiguous to me (although the name is pretty long then, so...)

Reply:
changed to `ctx_builder_...`